src/components/ReviewCard.jsx

```diff
@@ -8,7 +8,7 @@ const ReviewCard = ({ review }) => {
         <h2>{title}</h2>
         <h3>{designer}</h3>
         <p>{review_body}</p>
-      <Link to={`/reviews/${review_id}`}>
+      <Link to={`${review_id}`}>
```

Comment:
The "to" path here should be how it initially was: "/reviews/${review_id}" - did this not work? I've changed it to this in your code on my side and it seems to work fine. If it does work please change it.